apps/nextjs-website/src/lib/strapi/tutorial.ts

```diff
@@ -50,11 +50,11 @@ const makeStrapiTutorialsPopulate = () =>
       image: {
         populate: ['image'],
       },
-      parts: {
-        populate: ['backgroundImage'],
-      },
+      parts: '*',
```

Comment:
The build is failing  to collect page data for `/[productSlug]/tutorials/[...productTutorialPage]` and I think that the error is caused by the changes in this file. Why are you changing the populate here?

Reply:
i removed it because no parts (alert, codeBlock, embedHtml and htlm) have the backgroundImage (it was a parameter (optional) in the Quote part) 
<img width="754" alt="image" src="https://github.com/user-attachments/assets/4235f49a-d9cb-4530-8773-571f1122f110">